packages/infrastructure/authentik_atlas_mongodb_sso/README.md

```diff
@@ -0,0 +1,84 @@
+# Atlas MongoDB SSO with Authentik
+
+This module sets up the Authentik Application and Provider for Atlas MongoDB.
+
+## Sample Terragrunt Configuration
+
+```hcl
+include "panfactum" {
+  path   = find_in_parent_folders("panfactum.hcl")
+  expose = true
+}
+
+terraform {
+  source = include.panfactum.locals.pf_stack_source
+}
+
+dependency "authentik_core" {
+  config_path = "../authentik_core_resources"
+}
+
+dependency "kube_authentik" {
+  config_path = "../kube_authentik"
+}
+
+inputs = {
+  acs_url      = "https://auth.mongodb.com/sso/saml2/sample"
+  issuer       = "https://authentik.sample.com"
+  audience     = "https://www.okta.com/saml2/service-provider/sample"
```

Comment:
This doesn't seem correct?

Reply:
thought so  too but i believe mongo is using okta